iam/policy/admin-action.go

```diff
@@ -56,6 +56,8 @@ const (
 	HealthInfoAdminAction = "admin:OBDInfo"
 	// BandwidthMonitorAction - allow monitoring bandwidth usage
 	BandwidthMonitorAction = "admin:BandwidthMonitor"
+	// DownloadRawAction - allows downloading raw files from backend
+	DownloadRawAction = "admin:DownloadRaw"
```

Comment:
shouldn't this be something like InspectObject @klauspost ?

Reply:
@harshavardhana Dunno, it will not inspect anything, just download the file you ask for from all servers.